src/daemon/rpmostree-sysroot-upgrader.c

```diff
@@ -861,6 +861,18 @@ prep_local_assembly (RpmOstreeSysrootUpgrader *self,
   if (treespec == NULL)
     return FALSE;
 
+  RpmOstreeRefspecType refspec_type;
+  const char *refspec;
+  rpmostree_origin_classify_refspec (self->origin, &refspec_type, &refspec);
+  switch (refspec_type)
+    {
+    case RPMOSTREE_REFSPEC_TYPE_OSTREE:
+      break;
+    case RPMOSTREE_REFSPEC_TYPE_ROJIG:
+      /* We don't want to re-check the metdata, we already did that for the base */
+      rpmostree_context_set_pkgcache_only (self->ctx, TRUE);
```

Comment:
Hmm, is that what we want though?

`pkgcache_only` is more along the lines of "offline mode": it disables all the remote repos. I think we want more something like this `dnf_context_set_cache_age (dnfctx, G_MAXUINT)` here, right?

Reply:
Good catch.  Your suggestion would be right except we'd still get double output.  I pushed a fixup with a different approach :arrow_down: